grammar.js

```diff
@@ -1297,7 +1307,8 @@ module.exports = grammar({
 
     _reserved_identifier: $ => alias(choice(
       'default',
-      'union'
+      'union',
+      'await',
```

Comment:
Possible related to the conflict above. Why did you add this to `reserved_identifier`?

Reply:
Due to the nature of the await postfix syntax, it only appears after a expression. so it's use anywhere eslse should be of a identifier

Reply:
Yeah, I think you can remove this for that reason. This is only needed for tokens that can occur in expression or parameter position. I'm hoping that once you do this, you can remove the conflict that you added.